opm/autodiff/FullyImplicitBlackoilSolver_impl.hpp

```diff
@@ -1254,6 +1275,17 @@ namespace {
         struct Chop01 {
             double operator()(double x) const { return std::max(std::min(x, 1.0), 0.0); }
         };
+
+        double infinityNorm( const ADB& a )
+        {
+            if( a.value().size() > 0 ) {
+                return a.value().matrix().template lpNorm<Eigen::Infinity> ();
```

Comment:
exactly that, see http://en.cppreference.com/w/cpp/language/dependent_name near the end.